ui/packages/components/src/Forms/Input.tsx

```diff
@@ -14,7 +15,19 @@ const sizeStyles = {
 };
 
 export const Input = forwardRef<HTMLInputElement, InputProps>(
-  ({ className, type = 'text', inngestSize = 'base', ...props }, ref) => {
+  (
+    { allowPasswordManager = false, className, type = 'text', inngestSize = 'base', ...props },
+    ref
+  ) => {
+    let passwordManagerProps: Record<string, boolean> = {
+      'data-1p-ignore': true,
+      'data-bwignore': true,
+      'data-lpignore': true,
+    };
```

Comment:
I think to also disable Chrome's autocomplete for password management we need to add `autocomplete="off"` as well. Can we add that?